skimage/transform/_geometric.py

```diff
@@ -810,10 +775,6 @@ def __add__(self, other):
             else:
                 tform = ProjectiveTransform
             return tform(other.params @ self.params)
-        elif (hasattr(other, '__name__')
-                and other.__name__ == 'inverse'
-                and hasattr(get_bound_method_class(other), '_inv_matrix')):
-            return ProjectiveTransform(other.__self__._inv_matrix @ self.params)
```

Comment:
I considered this as obsolete.

The `elif` would check whether the passed `other` is not a transform instance but the `inverse` method object. Then it would get from the receiver (transform object) the inverse matrix attribute.

It would still be useful if someone was using multiple versions of scikit-image in the same code (so an transform inverse method could be added to a new transform object) or if we want to keep duck-typing-like compatibility to non-scikit-image transforms. I can revert it back.

Reply:
Ah, true. It checks whether the name of `other` is `inverse`; I actually read that as checking for having the attribute `inverse`. I'm onboard with removing it then. :+1: I don't think the mentioned remaining cases were ever intended to be part of the API.